cmd/snap-preseed/preseed_linux.go

```diff
@@ -152,6 +171,14 @@ func prepareChroot(preseedChroot string) (func(), error) {
 		}
 	}
 
+	// read version from the mounted core snap
+	infoPath := filepath.Join(snapdMountPath, dirs.CoreLibExecDir, "info")
```

Comment:
(nitpick) I understand it makes testing slightly easier but I think this Join should be done inside the function